nifi-provenance-reporting-tasks/src/main/java/io/egm/nifi/reporting/EmailProvenanceReporter.java

```diff
@@ -295,21 +329,30 @@ public void indexEvent(final Map<String, Object> event, final ReportingContext c
             getLogger().error("Error sending error email: " + e.getMessage(), e);
         }
     }
-
     public void sendErrorEmail(Map<String, Object> event, ReportingContext context) throws MessagingException {
 
-        String emailSubject = "Error occurred in processor " + event.get("component_name") + " "
-                + "in process group " + event.get("process_group_name");
+        String emailSubject = "[" + context.getProperty(EMAIL_SUBJECT_PREFIX).getValue() + "] : " + "Error occurred in processor " + event.get("component_name") + " "
+                + "in process group " + event.get("process_group_name") +   "from nifi instance: " ;
```

Comment:
`EMAIL_SUBJECT_PREFIX` is optional. If not defined, the subject will have an empty `[]` (and that does not look nice)